doc/man1/openssl-version.pod.in

```diff
@@ -77,8 +78,29 @@ The random number generator source settings.
 
 The OpenSSL CPU settings info.
 
+=item B<-w>
+
+The OpenSSL B<OSSL_WINCTX> build time variable, if set.
+Used for computing Windows registry key names
+
 =back
 
+=head1 HISTORY
+
+Previously, on Windows platforms, OpenSSL had a limitation regarding the
```

Comment:
Rather than "Previously" - perhaps we should say "In OpenSSL versions prior to 3.4"?